### PR TITLE
INTER-280 Add requirements section

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

**Review by ilfa — APPROVED** [review]

**Review by ilfa — CHANGES_REQUESTED** [review]
Could you add `pnpm` snippets here too?

**Review by ilfa — APPROVED** [review]

File: packages/fingerprintjs-pro-vue-v2/README.md [hunk 1]
```diff
@@ -30,8 +30,18 @@
```
[the remark this answers is not in the document]
Comment (JuroUhlar) [reply packages/fingerprintjs-pro-vue-v2/README.md:30]: Please address all the noticed issues also in the Vue 3 README consistently :)

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: packages/fingerprintjs-pro-vue-v2/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply packages/fingerprintjs-pro-vue-v2/README.md:30]: Done!

**Review by JuroUhlar — APPROVED** [review]
Looking good, thank you!

**Review by ilfa — APPROVED** [review]

**Pull request merged**